e2e/tests/meditor.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { test, expect, type Page } from "@playwright/test";
+import { clientUrl, gotoAndLogin } from "../utils.ts";
+
+const noLicenseSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(page.getByText("required information")).toBeVisible();
+};
+const noContributorSelector = async (page: Page) => {
+  await expect(
+    page.getByRole("tab", { name: "Dandiset contributors" }),
+  ).toBeVisible();
+};
+const invalidLicenseSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(page.getByText('must be equal to one of the')).toBeVisible();
+};
+const titleTooLongSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(
+    page.getByText("must NOT be longer than 150 characters"),
+  ).toBeVisible();
+};
+
+const dandisetsToTest = [
+  "000003",
+  "000018",
+  "000024",
+  // "000071",
+  "000106",
+  // "000107",
+  "000362",
+];
+
+const invalidDandisets = {
+  "000018": [noLicenseSelector],
+  "000024": [noContributorSelector, noLicenseSelector],
+  "000071": [noContributorSelector],
+  "000106": [invalidLicenseSelector],
+  "000107": [invalidLicenseSelector, noContributorSelector],
```

Comment:
`000071` and/or `000107` can be removed here, right? (I say and/or because of my previous comment about testing one of them)

Reply:
Yeah I agree a code comment would be appropriate here.

Reply:
Added